pkg/addons/embeddedclusteroperator/embeddedclusteroperator.go

```diff
@@ -76,9 +76,9 @@ func (e *EmbeddedClusterOperator) GenerateHelmConfig() ([]v1beta1.Chart, []v1bet
 	return []v1beta1.Chart{chartConfig}, nil, nil
 }
 
-// readEmbeddedClusterConfig reads and unmarshal the Config object from the embedded cluster
+// ReadEmbeddedClusterConfig reads and unmarshal the Config object from the embedded cluster
 // that has been embedded into this binary through a release.
-func (e *EmbeddedClusterOperator) readEmbeddedClusterConfig() (*embeddedclusterv1beta1.Config, error) {
+func (e *EmbeddedClusterOperator) ReadEmbeddedClusterConfig() (*embeddedclusterv1beta1.Config, error) {
```

Comment:
Couldn't you call `customization.AdminConsole{}.EmbeddedClusterConfig()` directly ?

Reply:
yeah but this does the parsing and I'd rather just have that in one place

investigating that is what led to my "we expect unsupported overrides to be two different formats" comment in slack

Reply:
I've updated `customization.AdminConsole{}.EmbeddedClusterConfig()` to return the parsed version, and removed this function entirely